lib/rex/post/meterpreter/extensions/stdapi/sys/process.rb

```diff
@@ -319,6 +319,35 @@ def path
     return get_info()['path']
   end
 
+  #
+  # Read data from the process channel
+  #
+  def read_data(time_out_start = 0, time_out = -1)
```

Comment:
Since this calls `close` on the command channel, maybe this should be named something like `capture_output` as a reference to the `Open3.capture*` methods?

Reply:
This sounds alot cleaner than what I was thinking. If this is always reading until close, that would also mean that the `time_out_start` parameter probably isn't necessary since this won't be called repeatedly. Right now both callers are just initializing it to the current time.